streetcrm/widgets.py

```diff
@@ -73,26 +73,36 @@ class TwelveHourTimeWidget(forms.MultiWidget):
         (12, "12"),
     )
 
+    MINUTE_CHOICES = (
+        ("", "--"), # Blank - no time specified.  This defaults to 00
```

Comment:
Why even have a "--" choice for minutes?  Just default to "00", no?

Reply:
So that it doesn't have "--, 00" as the default.  I guess I thought "--, --" looks better?

Reply:
Oh, I see.  Yes, that makes sense.